components/Accordion/AccordionItem/AccordionItem.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styles from './AccordionItem.css';
+
+AccordionItem.propTypes = {
+  title: string.isRequired,
+  content: node.isRequired,
+};
+
+function AccordionItem({ title, content }) {
+  // Create an HTML-safe ID for input query selecting
+  const titleID = title.replace(/[.,/#!?@#$%^&*;:{}=\-_'`~()]/g, '').replace(/\s{2,}/g, ' ');
+
+  return (
+    <div className={styles.accordionSingle}>
+      <input className={styles.accordionSingleHidden} type="checkbox" id={titleID} />
```

Comment:
You we're onto the right track with the abstraction! You'll just wanna get rid of references to `single` and `question` and `answer` in the CSS classnames.